tests/frame/to_pandas_test.py

```diff
@@ -22,10 +23,10 @@ def test_convert_pandas(
 ) -> None:
     data = {"a": [1, 3, 2], "b": [4, 4, 6], "z": [7.0, 8.0, 9.0]}
     df_raw = constructor_eager(data)
-    result = nw.from_native(df_raw).to_pandas()  # type: ignore[union-attr]
+    result = nw.from_native(df_raw, eager_only=True).to_pandas()  # type: ignore[union-attr]
```

Comment:
I can't seem to get `[union-attr]` to trigger here

```suggestion
    result = nw.from_native(df_raw, eager_only=True).to_pandas()
```

I have a feeling that we're not getting `[unused-ignore]` warnings with:
https://github.com/narwhals-dev/narwhals/blob/2e693e8c6827b3a7cb6bb39d5cee764bd7d5a461/pyproject.toml#L223